src/2_RTB/PianoDiProgetto/content.typ

```diff
@@ -0,0 +1,10 @@
+/*************************************/
+/*    INSERIRE SOTTO IL CONTENUTO    */ 
+/*************************************/
+
+#include "sections/Introduzione.typ" 
+#include "sections/AnalisiDeiRischi.typ" 
+#include "sections/ModelloDiSviluppo.typ" 
+#include "sections/Pianificazione.typ" 
+#include "sections/Preventivo.typ" 
+#include "sections/Consuntivo.typ" 
```

Comment:
commentare questa sezione in quanto vuota

Reply:
solo Pianificazione e Consuntivo sono vuote quindi le ho commentate.